### DISCUSSION TITLE
Support deep linking into package tabs

### DISCUSSION THREAD

**Discussion by WesSouza** [npm/feedback#70] [Website] [answered]
It would be nice to be able to directly link into one of the tabs of a package page, so if a user accesses a URL such as https://www.npmjs.com/package/npm#versions, they can land on the Versions tab immediately.

**Comment by MylesBorins** [comment, marked as answer]
@WesSouza there is currently deep linking into the readme e.g.

https://www.npmjs.com/package/npm#super-easy-install

I'm guess you would like to see that also working for items that are not part of the README?

**Reply by WesSouza** [reply]
Yes, to reach the tabs navigation (Readme, Explore, Dependencies, Dependents and Versions).

**Reply by MylesBorins** [reply]
You can link to the tabs with the active-tab attribute

https://www.npmjs.com/package/npm?activeTab=versions

Is that sufficient? I'd be concerned with using ID based navigation in case there is a duplicate between the active tab + something in the README

**Reply by WesSouza** [reply]
Ah, I didn't know about that (or noticed it is the href for the tabs already!)

Thanks.

**Reply by Ethan-Arrowood** [reply]
> I'd be concerned with using ID based navigation in case there is a duplicate between the active tab + something in the README 

IIRC this is an intentional web design decision for accessibility & screen readers so best to keep as is
